### ISSUE TITLE
Can someone provide a reliable source for LATEST_NET_G.PTH file?

### ISSUE THREAD

**Issue by Duemellon** [thygate/stable-diffusion-webui-depthmap-script#150]
The automated download link provided by this install:
 **_https://sfu.ca/~yagiz/CVPR21/latest_net_G.pth_**
 
 fails.
 It fails when I put it into my browser manually or without the http as well. Is there a more reliable source? Is that file hosted here & I don't see it?

**Comment by thygate** [comment]
Yes the server seems to be intermittent, I'm not sure licensing allows for me to mirror it.

I believe there was another post here mentioning an unofficial mirror, I will update this post when I find it..

**Comment by thygate** [comment]
https://wormhole.app/5q4A1#fDPs4Ki9IFRnZouW7vqaRg

**Comment by Duemellon** [comment]
SFU.CA still timing out for me. D/ling from the fielshare site now, thanx.

**Comment by Duemellon** [comment]
Unfortunately:

**_loading the model from ./models/pix2pix\latest_net_G.pth
unexpected EOF, expected 3377425 more bytes. The file might be corrupted.
All done._**


** re d/l'd --- must've had a corrupt save initially now it's doing work

**Comment by krakennovelo** [comment]
Hello! Can I get a link for the latest_net_G.pth file? Download just doesn't start from this link https://sfu.ca/~yagiz/CVPR21/latest_net_G.pth

**Comment by thygate** [comment]
https://wormhole.app/xjA5P#9ilCfgOWUC39xO_N1x5LUA

We really should look into making a mirror somehow ..
